### ISSUE TITLE
MinGW64 GCC compiler isn't installed

### ISSUE THREAD

**Issue by dprestegard** [m-ab-s/media-autobuild_suite#1640]
Fresh install of MABS on Windows 10 x64, I've tried several times from scratch and it dies every time like this:

```
-------------------------------------------------------------------------------
install msys2 base system
-------------------------------------------------------------------------------
-------------------------------------------------------------------------------
install 64 bit compiler
-------------------------------------------------------------------------------
-------------------------------------------------------------------------------

MinGW64 GCC compiler isn't installed; maybe the download didn't work
Do you want to try it again?

-------------------------------------------------------------------------------
try again [y/n]:
```

**Comment by 1480c1** [comment]
Please at least upload the `media-autobuild_suite.ini` file under the build folder. A snippet from the output of the bat file without any context nor obvious errors shown is barely any information to go off of.

**Comment by sabs21** [comment]
I actually have this exact same problem as well. Here is my INI file contents. I'm running Windows 10 x64 with a fresh install. Ran through powershell.

[compiler list]
msys2Arch=2
arch=3
license2=1
standalone=1
vpx2=1
aom=1
rav1e=1
dav1d=1
x2643=4
x2652=2
other265=2
svthevc=1
xvc=2
vvc=2
svtav1=2
svtvp9=1
flac=1
fdkaac=1
faac=2
exhale=2
mediainfo=1
soxB=1
ffmpegB2=1
ffmpegUpdate=3
ffmpegChoice=2
mp4box=1
rtmpdump=1
mplayer2=2
mpv=1
vlc=2
bmx=2
curl=1
ffmbc=2
cyanrip2=2
redshift=2
ripgrep=1
jq=2
jo=2
dssim=1
avs2=1
CC=2
cores=2
deleteSource=1
strip=1
pack=2
logging=1
updateSuite=2
timeStamp=2
ccache=2
noMintty=2

**Comment by dprestegard** [comment]
Agreed! Hence why I'm opening this issue. It would be good to put that note (about including the media-autobuild_suite.ini file in the issue) in the docs, since I wouldn't have known to do that :)

Incidentally I completely deleted the msys64 and local64 folders and for some reason this time it worked. No idea why that wouldn't have been the case the first time, I did it exactly the same way (ran the .bat file from an interactive powershell session). Seems the first time it just didn't download gcc.

**Comment by 1480c1** [comment]
@sabs21 is your issue resolved

**Comment by sabs21** [comment]
It might be. I didn't delete the msys64 or local64 folders and simply reran the bat. It started downloading packages in mintty so I guess it's working properly?

**Comment by TedlyTheTalker** [comment]
I kept getting the same error messages, seems like something wasn't installing due to certificate errors. 

I was able to work around it:
1. Delete local64 & mysys64 directories (presumably the 32-bit folders if that's part of your build target)
2. Install msys2 from https://www.msys2.org/ (default directory)
3. Copy the install directory (mysys64) to the root of autobuild suite
4. Operate media-autobuild_suite.bat as directed by your doctor
